docs/conf.py

```diff
@@ -89,7 +89,7 @@
 
 html_context = {
     'current_language': 'English',
-    'languages': (('English', '/latest'), ('Chinese', '/cn/latest')),
+    'languages': (('English', '/latest'), ('中文', '/cn/latest')),
```

Comment:
```suggestion
    'languages': (('English', '/latest'), ('简体中文', '/zh_CN/latest')),
```
Maybe we should use `zh_CN` to align the language code in the repo? I'm thinking of future possibilities of adding Traditional Chinese versions.

Reply:
As this affects the language switcher, let's first put everything in place as is. Previous versions use "cn"